pkg/services/featuremgmt/service.go

```diff
@@ -57,6 +60,15 @@ func ProvideManagerService(cfg *setting.Cfg) (*FeatureManager, error) {
 	// update the values
 	mgmt.update()
 
+	// Log the enabled feature toggles at startup
+	enabled := sort.StringSlice(maps.Keys(mgmt.enabled))
+	logctx := make([]any, len(enabled)*2)
+	for i, k := range enabled {
+		logctx[(i * 2)] = k
+		logctx[(i*2)+1] = true
```

Comment:
I think it'd be (optionally? maybe?) valuable to also log toggles which are present but disabled, but that's not necessary right away for this.